src/components/DeckMechanisms/index.js

```diff
@@ -20,6 +20,17 @@ export const FRIENDLY_CHANCES = {
   W16: DAWNSPARKS_HITS * DAWNSPARKS_STAYS,
 }
 
+const FROZEN_ENEMIES_AFTER = {
+  // Frosthexers
+  W2: [1, 2, 3, 3, 4],
+  // Moment's Peace
```

Comment:
```suggestion
  // Moment’s Peace
```

It just grinds my gears. 😅